functions/package.json

```diff
@@ -23,12 +24,16 @@
     "wanakana": "^3.1.0"
   },
   "devDependencies": {
-    "eslint": "^3.19.0",
-    "eslint-config-semistandard": "^11.0.0",
-    "eslint-config-standard": "^10.2.1",
-    "eslint-plugin-import": "^2.3.0",
-    "eslint-plugin-node": "^5.0.0",
-    "eslint-plugin-promise": "^3.5.0",
-    "eslint-plugin-standard": "^3.0.1"
+    "ava": "^1.4.1",
+    "c8": "^3.5.0",
```

Comment:
Why did you use c8 as opposed to https://github.com/avajs/ava/blob/fd73358d9ee85c3623a10acfebb5af88c51d628b/docs/recipes/code-coverage.md?

Reply:
See https://medium.com/the-node-js-collection/rethinking-javascript-test-coverage-5726fb272949, `c8` seems maintained by istanbul maintainer (the instrumentation package that `nyc` depends on).

And produce similar results for `node >= 10` but using v8 native coverage rather than code instrumentation.

I found the following ongoing issue: https://github.com/bcoe/c8/issues/72 but it looks like this is already resolve upstream.

Reply:
Thanks for the link -- it was an interesting read.

Looks like issue you referenced hasn't been backported to `node <10`. I think a few of our samples have `node 8`, so this can be a challenge if applied to other samples.

However, I do like the native coverage provided by Node and the referenced issue is an edge case, so I am OK with this change.

Reply:
My assumption was that developer would have a more recent version of node locally (>=10), than the one they deploy to (<=8). Is that the case in practice?